website/docs/quick-start.md

```diff
@@ -31,32 +31,37 @@ This sample illustrates how to patch containers using vulnerability reports with
     ```bash
     trivy image --vuln-type os --ignore-unfixed mcr.microsoft.com/oss/nginx/nginx:1.21.6
 
-3. Patch the image using the Trivy report. You will need to start `buildkitd` if it is not already running:
-
+3. Patch the image using the Trivy report.:
+
+    You'll need a buildkit instance to connect to.
+    By default copa will attempt to auto-connect to an instance in order:
+        1. Default docker buildkit endpoint
+        2. Currently selected buildx builder (see: `docker buildx --help`)
+        3. buildkit daemon at the default address `/run/buildkit/buildkitd.sock`
+    
+    If an instance doesn't exist or that instance doesn't support all the features copa needs the next will be attempted.
+    You may need to specify a custom address using the `--addr` flag. Here are the supported formats:
+
+    - `unix:///path/to/buildkit.sock` - Connect to buildkit over unix socket.
```

Comment:
We should keep the instructions for this. Should we create a seperate section? wdyt?